code/game/objects/items/antiques.dm

```diff
@@ -0,0 +1,209 @@
+#define NO_GOD 0
+#define GOD_CAUTERIZE 1
+#define GOD_MEND 2
+#define GOD_CLEANSE 3
+#define GOD_ELDRITCH 4
+#define GOD_STONE 5
+#define GOD_SIGHT 6
+#define GOD_MUTE 7
+#define GOD_PARALIZE 8
+#define GOD_FORCE 9
+#define GOD_NETHER 10
+#define GODS_MAX 10
+
+/obj/item/artifact
+	name = "antique"
+	desc = "An antique item of some sort..."
+	icon = 'icons/obj/fluff.dmi'
+	icon_state = "paperstack"
+	var/inUse = FALSE
```

Comment:
nitpick but in_use would be more consistent

Reply:
Something like this?